src/pp-annotator/form/AnnotationForm.tsx

```diff
@@ -1,13 +1,6 @@
-import React, { Component } from 'react';
-import { annotationPriorities} from '../consts.js'
-
-function sliceKeys(dictionary, keys) {
-    let result = {};
-    keys.forEach(function(key) {
-        result[key] = dictionary[key];
-    });
-    return result;
-}
+import * as React from 'react';
```

Comment:
da się skonfigurować TSa tak, żeby działał zwykły
```
import React from 'react';
```

jeśli w tym jest problem, to mogę poszukać

Reply:
nie jestem akurat dokładnie pewien jak to działa, dobrze by było jakbyś poszukał